src/.defaultsForValidator.js

```diff
@@ -65,7 +65,7 @@ const defaults = {
       'no_property_description': 'warning',
       'description_mentions_json': 'warning',
       'array_of_arrays': 'warning',
-      'inconsistent_property_type': 'warning',
+      'inconsistent_property_type': ['warning', 'code', 'default', 'type', 'value'],
```

Comment:
I would make a suggestion for a small tweak here. I think the list of keys to exclude should be their own list. Right now, it looks to me like these values are separate arguments with potentially different purposes. Since they are all meant to be grouped as a list though, I think it makes more sense to have them be one argument in list form, like:

```js
['warning', ['code', 'default', 'type', 'value']]
```